scripts/common/addresses.ts

```diff
@@ -72,7 +73,8 @@ const addresses = {
         AUTOMATION_BASIC_BUY_COMMAND: '0x2003dC19056bA986B7d10AB4704897d685DD62D9',
         AUTOMATION_BASIC_SELL_COMMAND: '0x2eCC5086CE10194175607d0D082fC27c3416693d',
         AUTOMATION_BOT_AGGREGATOR: '0xeb3c922A805FAEEac8f311E1AdF34fBC518099ab',
-        CONSTANT_MULTIPLE_VALIDATOR: '0x2f81e8330b476e8b5eca2a3302b902c8d0b7c6ef',
+        CONSTANT_MULTIPLE_VALIDATOR: '0x9A0CC19E5D1891479257Ac7863851917950E0eBd',
```

Comment:
why validator also changes?

Reply:
that is because I was too fast with pressing enter while reverse searching the terminal for command and deployed the validator for the second time. We might want to add a middle step to confirm deployments :)